frontend-react/src/components/Content/MarkdownLayout.tsx

```diff
@@ -0,0 +1,34 @@
+import { GridContainer } from "@trussworks/react-uswds";
+import { MDXProvider } from "@mdx-js/react";
+
+import { USSmartLink } from "../USLink";
+import { GridRow } from "../Grid";
+
+export interface MarkdownLayoutProps {
+    sidenav?: JSX.Element;
+    children?: React.ReactNode;
+}
+
+export function MarkdownLayout({ sidenav, children }: MarkdownLayoutProps) {
+    return (
+        <MDXProvider
+            components={{
+                a: USSmartLink,
+            }}
+        >
+            <GridContainer>
+                <GridRow>
+                    {sidenav ? (
+                        <nav
+                            aria-label="side-navigation"
+                            className="grid-col-auto"
```

Comment:
Should this take up a set number of columns rather than auto?  For example, the Product pages use four columns for the side nav and eight columns for the main content on tablet+ and just stack on smaller viewports.